README.md

```diff
@@ -27,12 +28,11 @@ Both for using an up-to-date version of Goblint or developing it, the best way i
 2. Continue using Linux instructions in WSL.
 
 ### Other
-* **[opam](https://opam.ocaml.org/packages/goblint/)**. Install [opam](https://opam.ocaml.org/doc/Install.html) and run `opam install goblint`.
-* **[devcontainer](./.devcontainer/).** Select "Reopen in Container" in VS Code and continue with `make` using Linux instructions in devcontainer.
-* **[Docker (GitHub Container Registry)](https://github.com/goblint/analyzer/pkgs/container/analyzer)**. Run `docker pull ghcr.io/goblint/analyzer:latest` (or `:nightly`).
-* **Docker (repository).** Clone and run `docker build -t goblint .`.
-* **Vagrant.** Clone and run `vagrant up && vagrant ssh`.
-
+- **[opam](https://opam.ocaml.org/packages/goblint/)**. Install [opam](https://opam.ocaml.org/doc/Install.html) and run `opam install goblint`.
+- **[devcontainer](./.devcontainer/).** Select "Reopen in Container" in VS Code and continue with `make` using Linux instructions in devcontainer.
+- **[Docker (GitHub Container Registry)](https://github.com/goblint/analyzer/pkgs/container/analyzer)**. Run `docker pull ghcr.io/goblint/analyzer:latest` (or `:nightly`).
+- **Docker (repository).** Clone and run `docker build -t goblint .`.
+- **Vagrant.** Clone and run `vagrant up && vagrant ssh`.
```

Comment:
GitHub still shows these lines as changed for some reason. Maybe there's a whitespace change that needs to be reverted?

Reply:
I'm pretty sure that this was something my VSCode linter did automatically by mistake by inserting some spaces here and there in the `README`. I noticed that I had commit https://github.com/adelavais/analyzer/commit/cf0d5c45cdd35ce23090d2c9b6f6a712ac24efdd which fixed some merge conflict of the `README` with upstream and I think the spaces might be stemming from there as well